public/app/features/alerting/unified/utils/url.ts

```diff
@@ -1,10 +1,14 @@
 import { config } from '@grafana/runtime';
 
-export function createUrl(path: string, queryParams?: string[][] | Record<string, string> | string | URLSearchParams) {
+export function createRelativeUrl(
+  path: string,
```

Comment:
Just an idea, what if we used templated type like `path: `/${string}` instead? It should force the function's users to always provide a path with a leading slash. I see the function automatically adds the leading slash now, but adding a slash gives you a slightly different result so maybe it's better to force the slash using types?

Reply:
I love this approach ❤️

Reply:
Added a commit with this approach, there is only [one place where typescript still complains](https://github.com/grafana/grafana/pull/90667/files#diff-0979422fa60152db34cd8fd05b18557ef04ccae254bf59153deb6da6a168e08dR55), but I think we are ok ignoring it